server/migrations/20240828081032-add-notifications-index.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addIndex("notifications", ["documentId", "userId"]);
```

Comment:
Adding too many columns to this index could be detrimental to performance. 
We already have an index on `event` - postgres query planner should be smart enough to use multiple indexes when necessary.